surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/SecurityUtils.java

```diff
@@ -81,4 +82,27 @@
             }
         }
     }
+
+    public static Field getField(final Class<?> source, final String name) {
+        Field declaredAccessibleField = AccessController.doPrivileged(new PrivilegedAction<Field>() {
```

Comment:
We can return directly here instead of assigning it here and returning on next statement. Also we can use lambda here

Reply:
Yeah, copy-paste from the arquillian-core